### DISCUSSION TITLE
The dev pitful

### DISCUSSION THREAD

**Discussion by matt40k** [dataplat/DataSaturdays#77] [Q&A]
I don't want to sound like a Debbie downer on everyone's hard work to pick up from the the blackhole that SQL PASS is leaving \ left but I can see a lot of effort in planning on build a events system. I appreciate we have a lot of talent in the community who can do amazing things but is this the correct path? Reinventing the wheel. Just thinking we had days (not months\years) before PASS completely disappears. Wouldn't it be better to use something like [Indico](https://getindico.io/)? Then ideally pick a managed provider like [Mythic Beasts](https://www.mythic-beasts.com/apps/indico) (who help with the Raspberry Pi and **could** do a discount) to manage it - we can then review in say a year time, move hosting providers, redesign or whatever. So we can focus on content rather than the tool.

**Comment by way0utwest** [comment]
Using a provider involves contracts, payments, etc. Not necessarily a problem, but a dependency I would prefer to avoid. 

I am not opposed to using services, like Paypal, Sessionize, EventBrite,etc., but I do prefer to have some choice here, especially as we are not a central org, but trying to allow different organizers and geographies to diverge.

Using something like indico, with plugins, isn't something I'm opposed to. In fact, if it gets some things easier and we build plugins for items that are important to us, that's fine with me.

I think a committee to evaluate this is important. I think PASS failed by not building or using enough software on their own and passing on too many costs to other providers over the years. We want to fund things minimally here. I think the discussion of #55 might mean that we have an offering for some organizers, but providing some tools for others and allowing them flexibility is important.

The idea of using something quick likely is more of a #55 discussion than a technical one.
